### DISCUSSION TITLE
V3.2.3 is out ...

### DISCUSSION THREAD

**Discussion by thomasleitner** [doitandbedone/ispyagentdvr-docker#190] [General]
Hi, 

Looks as if V3.2.3.0 is out on https://www.ispyconnect.com/download.aspx ... it contains an important fix for me.
Would it be possible for you to create the docker version in the near future?

Thanks a lot ...

**Comment by doitandbedone** [comment]
Should be building now. Will be available in a few minutes.

**Reply by thomasleitner** [reply]
Woow ... great. Thanks a lot ...

**Comment by doitandbedone** [comment]
The versions gets tagged and built automatically. The actual images may be
a bit behind since they need manual review. So you can always target the
specific version if you want it earlier. I also noticed there's even a
newer version. Should be ready now though.
